src/authorizations/utils/permissions.ts

```diff
@@ -49,7 +49,9 @@ export const allAccessPermissions = (
   userID: string
 ): Permission[] => {
   const withOrgID = ensureT(orgID, userID)
-  return allPermissionTypes.flatMap(withOrgID)
+  return allPermissionTypes
+    .filter(perm => String(perm) !== 'instance')
```

Comment:
The reason for converting the type to a string is that the instance type does not exist at build time in a cloud build, so you get the `this line will always be true` linting error.

If that would be preferred over this, happy to change it or if there is a better way to do this in general.